OneCommunityGlobal/HGNRest
# Jiadong - Replace Badges Section on Dashboard / badge count APIs

# Description
![image](https://github.com/OneCommunityGlobal/HGNRest/assets/65985986/e21490a4-b263-4917-83cc-c356539c81f3)
![image](https://github.com/OneCommunityGlobal/HGNRest/assets/65985986/34179b15-59b9-4a95-9013-be41623b73fe)

## Related PRS (if any):
This frontend PR is related to the #[2146 backend PR.](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/pull/2146/)
…

## Main changes explained:
Add badge count APIs
…

## How to test:
check into current branch and frontend [2146](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/pull/2146/)
see frontend instruction

## Screenshots or videos of changes:

https://github.com/OneCommunityGlobal/HGNRest/assets/65985986/e0e974ab-ca4b-4fcd-b868-69f177cacc2d
